### DISCUSSION TITLE
Chord for easyclick macro or macro for typing two letter ref w/ easyclick

### DISCUSSION THREAD

**Discussion by DCurtin** [pit-ray/win-vind#227] [General]
Hello,
I was recently experimenting w/ macros in win-vind, trying to either have a macro fire for a particular sequence of characters or tie in some RPA to w/ wind-vind to do this.

I couldn't figure out a way of tying a two letter ref after <easyclick> in a macro.

for some context, there was a repetitive action in windows that I wanted to loop, every time wind-vind entered easyclick mode the same UI element always had the same two letter ref, so I could hard-code this for a one-off macro.

if I could have a macro accept something like this
imap <ctrl-3> <to_gui_normal><easyclick>{AT}
then that would have solved part of my problem.

**Comment by pit-ray** [comment]
Are you saying you want to select EasyClick hints with a macro?
